### DISCUSSION TITLE
MAC Client registration failed

### DISCUSSION THREAD

**Discussion by Leeero** [dani-garcia/vaultwarden#3117] [Q&A]
I need your help, thanks
use docker install bitwardenrs/server
bitwardenrs version：2.25.1
OS Client：2022.11.0 (5788)
error:
<img width="301" alt="image" src="https://user-images.githubusercontent.com/22819077/211004662-125167f3-6f52-46a9-90d1-3c519cdbc9d6.png">
error log :
<img width="464" alt="image" src="https://user-images.githubusercontent.com/22819077/211004758-c7e9489a-0945-4cb1-af6c-f2ace2939c63.png">

**Comment by stefan0xC** [comment]
Update vaultwarden to 1.27.0.
See #3082

**Comment by Leeero** [comment]
> 

thank you very much，The problem is indeed caused by the version。
This command should be used to pull the latest version of the image：docker pull bitwardenrs/server:testing
But I use： docker pull bitwardenrs/server:latest
bitwardenrs/server:testing ->1.27.0
bitwardenrs/server:latest ->1.24.0
Thanks again

**Comment by BlackDex** [comment]
You need to use `vaultwarden/server` instead of `bitwardenrs/server`. `bitwardenrs` is not updated any more since we moved to an other name
